index.d.ts

```diff
@@ -254,6 +254,12 @@ export declare interface TracerOptions {
      * @default []
      */
     peers?: string[]
+
+    /**
+     * A string representing the minimum tracer log level to use when debug logging is enabled
+     * @default null
+     */
+    logLevel?: string
```

Comment:
The accepted values should be listed. For example, right now it would be `error` and `debug`, but we plan to also add a few more levels.

Reply:
updated to specify accepted values (`error` or `debug`)